waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -542,6 +542,10 @@ proc queryWithAccounting*(ws: WakuStore, query: HistoryQuery, handler: QueryHand
   let peerId = peerOpt.get().peerId
   let messages = response.value.response.messages
   ws.wakuSwap.debit(peerId, messages.len)
+  
+  #log the swap account balance
+  for peerId, bal in ws.wakuSwap.accounting:
+    debug "Accounting: ", peer=peerId, balance=bal
```

Comment:
If the logic here is to log all the balances after every `debit()`, shouldn't it be done after every [`credit()`](https://github.com/status-im/nim-waku/blob/37eb9e977c6ccadb53ba03efeaad69ebb376bc49/waku/v2/protocol/waku_store/waku_store.nim#L363) as well?